language-plutus-core/src/Language/PlutusCore/Name.hs

```diff
@@ -20,14 +20,14 @@ import           Data.Text.Encoding   (decodeUtf8)
 import           PlutusPrelude
 
 -- | A 'Name' represents variables/names in Plutus Core.
-data Name a = Name { attribute :: a
-                   , asString  :: BSL.ByteString -- ^ The identifier name, for use in error messages.
-                   , unique    :: Unique -- ^ A 'Unique' assigned to the name during lexing, allowing for cheap comparisons in the compiler.
+data Name a = Name { nameAttribute :: a
+                   , asString      :: BSL.ByteString -- ^ The identifier name, for use in error messages.
```

Comment:
Why not change `asString` to `nameString` to name all fields consistently?

Reply:
Will do!